docs/src/content/docs/guides/components.mdx

```diff
@@ -296,3 +269,79 @@ The code above generates the following on the page:
 import importedCode from '/src/env.d.ts?raw';
 
 <Code code={importedCode} lang="ts" title="src/env.d.ts" />
+
+### FileTree
+
+Use the `<FileTree>` component to display a file tree in your documentation representing the structure of a directory.
+
+A `<FileTree>` requires an [unordered Markdown list](https://www.markdownguide.org/basic-syntax/#unordered-lists) of files and directories.
+Directories can either end with a `/` or contain a nested list of files and directories.
```

Comment:
Suggesting a small rewrite here to flip the order to start with “user needs” rather than the more passive “component needs”.

```suggestion
Specify the structure of your files and directories with an [unordered Markdown list](https://www.markdownguide.org/basic-syntax/#unordered-lists) inside `<FileTree>`.
Create a sub-directory using a nested list or add a `/` to the end of a list item to render it as a directory without specific content.
```